index.html

```diff
@@ -52,114 +52,120 @@ <h1>Personal Profile</h1>
+                    <p>
+                        <ul>
+                            <li>React, jQuery, Flask, Firebase, Leaflet, Bootstrap</li>
+                        </ul>
                     </p>
                 </article>
                 <article>
-                    <h2>Freelance Web Design</h2>
-                    <p class="subDetails">2016 - 2017</p>
-                    <p><ul>
-                        <li>Consulted on design and management of online presence for small retail business.</li>
-                    </ul></p>
+                    <h2>Other</h2>
+                    <p>
+                        <ul>
+                            <li>Git</li>
+                        </ul>
+                    </p>
                 </article>
+            </div>
+            <div class="clear"></div>
+        </section>
+        
+        <section>
+			<div class="sectionTitle">
+				<h1>Education</h1>
+			</div>
+			
+			<div class="sectionContent">
                 <article>
-                    <h2>Tutor</h2>
-                    <p class="subDetails">2011 - 2013</p>
+                    <h2>freeCodeCamp</h2>
+                    <p class="subDetails">January 2019 - present</p>
                     <p><ul>
-                        <li>Tutored students at the Vrije Universiteit in Math and English.</li>
-                    </ul></p>
+                          <li>Responsive Web Design Certificate</li>
+                          <li>JavaScript Algorithms and Data Structures Certificate</li>
+                        </ul>
+                    </p>
+                    
+                </article>
+                <article>
+                    <h2>EDX Free Online Courses</h2>
+                    <p class="subDetails">July 2016 - July 2018</p>
+                    <p>
+                      <ul>
+                          <li>CS50's Introduction to Computer Science - HarvardX</li>
+                          <li>Introduction to Computer Science and Programming Using Python - MITx</li>
+                      </ul>
+                    </p>
                 </article>
 				<article>
-					<h2>Registered Nurse at Covenant Medical Center, Lubbock, TX</h2>
-					<p class="subDetails">2009 - 2010</p>
-					<p><ul>
-                           <li>Independently provided safe, therapeutic, and efficient care for a group of patients on a renal-telemetry
-                               step-down unit.</li>
-                           <li>Documented patient care services in department records.</li>
-                           <li>Collaborated with physicians and multidisciplinary team members and provided psychological support to patients
-                               and their families.</li>
-                    </ul></p>
+					<h2>Vrije Universiteit Amsterdam, Amsterdam, The Netherlands</h2>
+                    <p class="subDetails">August 2011 - May 2013</p>
+					<p>Completed 87 credits towards a Bachelor of Science in Mathematics.</p>

```

Comment:
If you did any impressive-sounding courses here (e.g. not just "calculus") call them out.

Symbolic logic? Set/Group theory? Optimization? Discrete math? Stats?